e2e-tests/build.gradle

```diff
@@ -54,12 +57,17 @@ task e2eTest(type: Test) {
 }
 
 dependencies {
+  testAnnotationProcessor project(':procedural:processor')
```

Comment:
I actually don't like calling these examples. They are a little too simplistic and unrealistic, and I wrote them for testing anyway. These are basically just assets for the e2e tests, and its normal to put test assets in the same package.

Reply:
That's fair, although I do think it would be fruitful to surface some useful example procedures in the top level `examples` folder eventually, but maybe that can result from our eventual monorepo w/ example mission models / schedulers in the future. Certainly not a blocker for this PR, happy to table it.